src/failures/src/test-values.ts

```diff
@@ -0,0 +1,60 @@
+import { FailuresOrchestrator, RemoteFailuresProvider } from '.';
+import { getActivateFailureSimVarName, getDeactivateFailureSimVarName } from './sim-vars';
+import { FailuresProvider } from './failures-provider';
+
+export function flushPromises() {
+    return new Promise((resolve) => setImmediate(resolve));
+}
+
+const prefix = 'PREFIX';
```

Comment:
**thought**: I'd be very careful about putting all these items in one file. I deliberately chose not to, because understanding tests where everything can be found locally is easier.
**suggestion**: Move code back to their original position.